sdk/src/main/java/com/oursky/authgear/AppInitiatedSSOToWebNotAllowedException.kt

```diff
@@ -0,0 +1,12 @@
+package com.oursky.authgear
+
+sealed class AppInitiatedSSOToWebNotAllowedException : AuthgearException {
```

Comment:
Do not use sealed class in public API. I do not know how is it translated to Java. Just use normal class hierarchy to represent these exceptions.

Reply:
https://medium.com/asos-techblog/kotlin-classes-in-java-world-365323843e6e

It will be something like

```java
abstract class PreAuthenticatedURLNotAllowedException {
  static final class InsufficientScopeException extends PreAuthenticatedURLNotAllowedException {
  }
  static final class IDTokenNotFoundException extends PreAuthenticatedURLNotAllowedException {
  }
  static final class DeviceSecretNotFoundException extends PreAuthenticatedURLNotAllowedException {
  }
}
```

Reply:
But exception as a inner class in uncommon. I still prefer we use regular classes.

Reply:
@louischan-oursky Added a commit to change it to regular classes.